test/controllers/questions_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class QuestionsControllerTest < ActionController::TestCase
+  setup do
+    @question = questions(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:questions)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create question" do
+    assert_difference('Question.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.